### DISCUSSION TITLE
Transferred issues stuck in loop

### DISCUSSION THREAD

**Discussion by qreeves** [community/community#2985] [Discussions]
My project, Red Eclipse, previously used issues on a separate repository (`redeclipse/world`) for discussions, to keep the clutter out of the issues on the main repository (`redeclipse/base`). With the introduction of GitHub discussions, I have attempted to migrate from there to the new discussions on the main repository, after which all the discussions are stuck in a loop ("*This discussion is being transferred. The timeline may not be complete until it finishes."*) despite the migration appearing to have completed successfully.

Workflow:
- Enable Discussions on `redeclipse/world`.
- Use "Convert Issues" in the label editor to migrate all issues to a discussion.
- Enable Discussions on `redeclipse/base`.
- Transfer each discussion one by one from `redeclipse/world` to `redeclipse/base`.

While a simple warning message in itself wouldn't be particularly bothersome, it makes the discussion area freak out when you view one of the affected discussion pages ([example](https://github.com/redeclipse/base/discussions/1218)). Basically, once you view one, the browser starts constantly refreshing the page and any attempt to navigate away will just bring you straight back to the discussion on the next refresh iteration.

As it is, all migrated discussions are practically unusable for our project at the moment, and it'd be appreciated if some priority was given to this issue.

**Comment by becca** [comment]
We are looking into this issue, thank you for letting us know. We'll do our best to keep you posted on when it's resolved.

**Reply by qreeves** [reply]
Is there an ETA on this? It creates some serious usability issues.

**Reply by evi-liu** [reply]
Hi @qreeves, our team is currently working on this bug. We're hoping to get a fix out for this by end of week.

**Reply by qreeves** [reply]
Alright, thank you 👍

**Reply by qreeves** [reply]
It has now been 14 months and our migrated discussions still show as *"This discussion is being transferred. The timeline may not be complete until it finishes."*

**Reply by ctrlplusb** [reply]
Oh dear. We've just joined the "This issue is being transferred. Timeline may not be complete until it finishes." gang. Not for discussions, but approximately 30% of the issues we are transferring have resulted in this message.

**Reply by qreeves** [reply]
> Oh dear. We've just joined the "This issue is being transferred. Timeline may not be complete until it finishes." gang. Not for discussions, but approximately 30% of the issues we are transferring have resulted in this message.

Yeah I don't think they care.
